tests/untestable/gdb/gdb_pretty_printer_test_output.txt

```diff
@@ -1,3 +1,3 @@
 Loading Nim Runtime support.
-NimEnumPrinter: lookup global symbol 'NTI_z9cu80OJCfNgw9bUdzn5ZEzw_ failed for tyEnum_MyOtherEnum_z9cu80OJCfNgw9bUdzn5ZEzw.
```

Comment:
Why does this JSON PR have changes to gdb?

Reply:
Yea I know it isn't clean. But I used gdb and fixed it along the way. Then I was too lazy to split the PR up into different parts as the changes in gdb and json were interleaved.